src/waveform/renderers/waveformrenderplaymarker.cpp

```diff
@@ -38,4 +56,36 @@ void WaveformRenderPlayMarker::draw(QPainter* painter, QPaintEvent* event) {
         painter->drawLine(0, lineY + 1, width, lineY + 1);
         painter->drawLine(0, lineY - 1, width, lineY - 1);
     }
+
+    // Draw bar and beat counter
+    if (prevBeat != mixxx::kInvalidBeat) {
+        QString barBeatString = QString("%1.%2").arg(
```

Comment:
```suggestion
        QString barBeatString = QString("%1:%2").arg(
```
IMO the use of `.` to separate bars and beat numbers is confusing. It looks like one number with a decimal point. I propose using `:` as the separator instead.

Reply:
I just took some inspiration from the Ableton grid. I would change it if it makes it more clear.

Reply:
@Be-ing this is what the Ableton beat and bar numbers look like:
![55a6747deef35](https://user-images.githubusercontent.com/30816844/90533242-d8afd480-e195-11ea-9d9e-5aeebe6843ad.jpg)

Reply:
I don't have a strong opinion either way. IIRC Rekordbox also uses the `bar.beat` notation. Does anyone else have thoughts on this?

Reply:
I suggest adopting the common notation that users are familiar with instead of re-inventing the wheel. The formatting should be moved into a public (member) function for reusability.

Reply:
okay